public/js/index.js

```diff
@@ -129,6 +129,32 @@ function handleViewClick(lesson) {
 	overlay.classList.add('dark');
 }
 
+function lessonHelper(varName, eventListener, classList, attribute, textContent, innerHTML, id) {
```

Comment:
Nice work! This works well. Another way to implement this would be to pass a single object to `lessonHelper`. It could also create the element too - something like this:

```
lessonHelper({
  type: 'div',
  classlist: ['test'],
  ... etc
})
```
You could then use JS object destructuring to pull the variables you need. i..e

```
function lessonHelper({ type, classlist, ... }) {
 ....
}
```
That could be for another time 😄

Reply:
Thank you and this is a great idea. Let me implement it real quick and get it push up in this PR. Thank you @jrobind .